src/readwrite.rs

```diff
@@ -104,6 +112,35 @@ impl<'env, K> Reader<'env, K> where K: AsRef<[u8]> {
     pub fn abort(self) {
         self.tx.abort();
     }
+
+    pub fn iter<'s>(&'s self) -> Result<Iter<'s>, StoreError> {
```

Comment:
This is actually equivalent to `iter_start` on `RoCursor`, so perhaps use that name for clarity?

Reply:
Good point! I've done so in 7a0d338.